tracer/test/snapshots/AwsLambdaTests.verified.txt

```diff
@@ -30,13 +30,13 @@ at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, Cancellat
     }
   },
   {
-    TraceId: Id_1,
-    SpanId: Id_4,
+    TraceId: Id_4,
```

Comment:
Just to make sure I understand this correctly, each span now has a new trace-id and parent-span-id because the dockerfile for the "extension" is not returning an active trace-context. Is that right?

Also, how difficult would it be to test a second endpoint that does provide an active trace-context? It would be good to test both scenarios in our automated E2E tests

Reply:
✅

Reply:
(we can now see that all three tests withContext share the same trace id (7 in the snapshot)